memdx/ops_crud.go

```diff
@@ -1300,7 +1300,7 @@ func (o OpsCrud) GetMeta(d Dispatcher, req *GetMetaRequest, cb func(*GetMetaResp
 			Value: resp.Value,
 			Cas:   resp.Cas,
 		}
-		res.Deleted = binary.BigEndian.Uint32(resp.Extras[0:])
+		res.Deleted = binary.BigEndian.Uint32(resp.Extras[0:]) == 1
```

Comment:
This should be != 0, since the server might want to add support for "deleted flags" or something.